plugins/inputs/t128_metrics/t128_metrics_test.go

```diff
@@ -480,17 +487,22 @@ func TestTimoutUsedForRequests(t *testing.T) {
 			map[string]string{"test-field": "stats/test"},
 			map[string][]string{},
 		}},
-		Timeout:                 internal.Duration{Duration: 5 * time.Millisecond},
+		Timeout:                 internal.Duration{Duration: 1 * time.Millisecond},
 		MaxSimultaneousRequests: 1,
 	}
 
 	var acc testutil.Accumulator
 	require.NoError(t, plugin.Init())
 
 	require.NoError(t, plugin.Gather(&acc))
+	done <- struct{}{}
+
 	require.Len(t, acc.Errors, 1)
+	require.EqualError(
+		t,
+		acc.Errors[0],
+		fmt.Sprintf("failed to retrieve metric stats/test: Post %s/stats/test: net/http: request canceled (Client.Timeout exceeded while awaiting headers)", fakeServer.URL))
 
-	fakeServer.CloseClientConnections()
```

Comment:
No longer needed? If anything, I think there definitely will be client connections still since the two require statements above will be fast and you are waiting for 10s

Reply:
It shouldn't be needed. The done channel will close the connections quickly.